doc/api/dgram.md

```diff
@@ -283,6 +283,9 @@ not work because the packet will get silently dropped without informing the
 source that the data did not reach its intended recipient.
 
 ### socket.setBroadcast(flag)
+<!-- YAML
+added: v0.6.9
```

Comment:
@bnoordhuis I got v0.6.9 from f749338e1eee390fc654f129f4419895db4f8e50. 

In v0.6.8, the function existed, but this was the implementation:

```js
Socket.prototype.setBroadcast = function(arg) {
  throw new Error('not yet implemented');
};
```

My interpretation was that the function existed in earlier releases, but went away for a while, and then came back in 0.6.9. Am I mistaken? Was that function never really run and there was a version in `dgram_legacy.js` or something that was actually the implementation for 0.6.8?